SwitchSpeak-iOS/SwitchSpeak-iOS/TouchGrid/ButtonNode.swift

```diff
@@ -14,21 +14,32 @@ import UIKit
 */
 class ButtonNode: Node {
 	var button = UIButton(type: UIButtonType.custom)
+    var dummy = false
 	
 	init(button: UIButton) {
 		self.button = button
 		super.init()
 	}
 	
 	override func highlightSubTree() {
+        if dummy==true{
```

Comment:
Couldn't this just be `if dummy == true { ... }`?

Reply:
it can not,  since the value of optional type "bool?"  not unwrapped